fuel/src/test/kotlin/com/github/kittinunf/fuel/BaseTestCase.kt

```diff
@@ -1,24 +1,3 @@
 package com.github.kittinunf.fuel
 
-abstract class BaseTestCase {
-
-    init {
-        enableTestMode()
-    }
-
-    protected inline fun regularMode(runnable: () -> Unit) {
-        disableTestMode()
-        runnable()
-        enableTestMode()
-    }
-
-    protected fun enableTestMode() {
-        Fuel.testMode {
-            timeout = 15000
-        }
-    }
-
-    protected fun disableTestMode() {
-        Fuel.regularMode()
-    }
-}
+abstract class BaseTestCase
```

Comment:
Maybe we don't need this anymore? 🔥 ?

Reply:
Is this removing? Why does Github shows as 🍏which looks like you change to have only this line? If you remove it, I guess it is okay.

Reply:
Just didn't delete the file yet 🗑 🗡

Reply:
♻️

Reply:
Done